test/unit/alphabet/semi_alphabet_test_template.hpp

```diff
@@ -0,0 +1,187 @@
+// -----------------------------------------------------------------------------------------------------
+// Copyright (c) 2006-2019, Knut Reinert & Freie Universität Berlin
+// Copyright (c) 2016-2019, Knut Reinert & MPI für molekulare Genetik
+// This file may be used, modified and/or redistributed under the terms of the 3-clause BSD-License
+// shipped with this file and also available at: https://github.com/seqan/seqan3/blob/master/LICENSE.md
+// -----------------------------------------------------------------------------------------------------
+
+#include <gtest/gtest.h>
+
+#include <seqan3/alphabet/concept.hpp>
+#include <seqan3/alphabet/exception.hpp>
+#include <seqan3/core/concept/core_language.hpp>
+// #include <seqan3/test/pretty_printing.hpp>
```

Comment:
artefact?

Reply:
ops. I think this caused problems with the mask alphabet because it is no debug_streamable. I might need to change that